docs/usage/nuget.md

```diff
@@ -98,9 +98,12 @@ Credentials for authenticated/private feeds can be given via host rules in the c
 }
 ```
 
+When using Azure DevOps, you can use `pkgs.dev.azure.com` as host name.
```

Comment:
```suggestion
When using Azure DevOps, you can use `pkgs.dev.azure.com` as `matchHost` value.
```

Is this what you meant?

Reply:
Correct - Didn't want to use the config name but that's fine for me